pdns/nsecrecords.cc

```diff
@@ -76,9 +76,8 @@ void NSECBitmap::toPacket(DNSPacketWriter& pw)
 {
   NSECBitmapGenerator nbg(pw);
   if (d_bitset) {
-    size_t count = d_bitset->count();
     size_t found = 0;
-    for(size_t idx = 0; idx < nbTypes && found < count; ++idx){
+    for(size_t idx = 0; idx < nbTypes && found < d_bitset->count(); ++idx){
```

Comment:
`std::bitset::count()` is likely O(n) and I'm not 100% confident that the compiler will be able to cache it, since the body of the loop calls functions that, as far as the compiler know, could possibly alter the bitset.